### ISSUE TITLE
Add discord widget label

### ISSUE THREAD

**Issue by SimplyPancake** [Inter-Actief/amelie#677]
The DisCo (Discord comittee) would like a discord widget for the Inter-Actief website.
For this, we could use the already provided iframe widget:

```html
<iframe src="https://discord.com/widget?id=613755161633882112&theme=dark" width="350" height="500" allowtransparency="true" frameborder="0" sandbox="allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts"></iframe>
```

The widget theme can also be set to light if that matches the website better.
If needed, the server ID is `613755161633882112` and the JSON API is `https://discord.com/api/guilds/613755161633882112/widget.json`

**Comment by SimplyPancake** [comment]
I would be fine with picking up this issue whenever I'm done with others

**Comment by Kurocon** [comment]
Am I assuming correctly that this would be  a new widget on the main page of the website? Or am I misunderstanding?

If the first case, I would recommend the widget to be visible only for logged in users and in the light theme.

**Comment by SimplyPancake** [comment]
Yes, this would be intended for the home page and indeed for only logged in members!

**Comment by WKobes** [comment]
We should check if the widget sets any cookies though, since otherwise we have to ask permission

**Comment by SimplyPancake** [comment]
It looks like discord is not setting any cookies. They are also not mentioning their widget in their privacy statement and they have included the following information in [one of their blogposts](https://support.discord.com/hc/en-us/articles/360039598252-Protecting-Your-Data): `We've anonymized and limited data returned on server widgets
`
It looks like they are also conscious about this issue, so I think it is fair to say that we can include the widget in the website

**Comment by Kurocon** [comment]
See #721 

The widget actually does set two third party cookies on the discord.com domain, which is currently not allowed by our privacy policy. So we have to revert this for now.

We can look in to changing the policy, which would require us to add a cookie banner (I would vote against this), or try to use the discord API to build our own 'widget' with a similar view, that does not use cookies.
